api/service/dashboard/dashboard.go

```diff
@@ -17,17 +17,18 @@ type dashboard struct {
 }
 
 var chartCriteriaByDuration = map[Duration]struct {
-	Ago     string
-	TruncBy time.Duration
+	Ago          string
+	TruncByInSql string
+	TruncBy      time.Duration
```

Comment:
nit. You can remove it if TruncBy is not used

Reply:
[a50f46f](https://github.com/dezswap/dezswap-api/pull/37/commits/a50f46fe2c9c930a914f8eeecabd089eca9e1974)